docs/TESTING.md

```
@@ -0,0 +1,93 @@
+
+To run tests in watch mode:
+
+```bash
+npx jest --watch
+```
+
+## Directory Structure
+
+Tests are located alongside the source code in `__tests__` directories, following the convention:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)